### DISCUSSION TITLE
Add variables support for publish panel

### DISCUSSION THREAD

**Discussion by mliubimov** [foxglove/community#654] [Q&A]
Now it seems like the publish panel does not support variables. Therefore, it is impossible to send a value that was modified by a slider, for example.
I think it is possible to add something with this syntax: 
`{
"field": $variable
}`
I think this could help to create more advanced logic for the UI.

**Comment by Onicc** [comment]
I have the same problem; I don’t even know what the slider is for.

**Comment by jtbandes** [comment]
Internal tracking issue: FG-8928

**Comment by AndyMRoberts** [comment]
same, this would be great
